spring-boot/src/main/java/org/springframework/boot/logging/logback/SpringPropertyAction.java

```diff
@@ -32,10 +32,12 @@
  * properties to be sourced from the Spring environment.
  *
  * @author Phillip Webb
+ * @author Eddú Meléndez
  */
 class SpringPropertyAction extends Action {
```

Comment:
More importantly than the comment below, these changes need to be made on `SpringProfileAction`, not `SpringPropertyAction`.